app/Lib/Export/NidsExport.php

```diff
@@ -16,11 +16,14 @@ class NidsExport
 				'fields' => array('threat_level_id')
 			)
 		),
-		'flatten' => 1
+		#'flatten' => 1
```

Comment:
Any debugging / test commented lines should be removed

Reply:
Done.